### DISCUSSION TITLE
iOS and/or iPad Support

### DISCUSSION THREAD

**Discussion by thismarty** [flxzt/rnote#1127] [Q&A] [answered]
Any chance rnote will eventually be built for iOS/iPad?

NOTE: I noticed that there is a dev doing macOS builds.  Since the Mac isn't exactly a good match for a 'pen first' note-taker like rnote, I was hoping that those macOS builds are a stopgap to test and distribute XCode rnote builds in anticiaption of an eventual iOS/iPadOS build when rnote hits 1.0.

**Comment by Doublonmousse** [comment, marked as answer]
Well, the toolkit used for the ui (gtk 4) doesn't support mobile (only linux, windows and mac) and afaik has no plans to extend support to mobile. See #710.

For now the renderer depends on gtk so that also prevents porting to other platforms by only rewriting the ui part and keeping the same engine code. Additionally, I'm not sure either every dependency would compile on ios at the moment (there's some rust dependencies that are wrappers around C/C++ libraries and these may not work or compile on ios).

So there won't be an ios version in the short term.

Maybe it will become more feasible in the long run though.

**Comment by thismarty** [comment]
Thanks for the insights!  I sure hope it gets easy enough to do down the road.  In the meantime, thanks very much for taking the time to do the macOS builds!
